content/client-lib-development-guide/features.textile

```diff
@@ -2015,6 +2050,59 @@ enum StatsIntervalGranularity:
   DAY
   MONTH
 
+class Stats.ConnectionTypes // TS4
```

Comment:
I'm not sure about this so feel free to dismiss: for consistency with the rest of the IDL this should probably just be `class ConnectionTypes`, although that may be a bit confusing so maybe we do something like `class ConnectionTypesStats`, wdyt?

Reply:
I think that I'd prefer to keep the naming consistent with that used in the feature spec, which uses these namespaced `Stats.` types. I think if we wanted to change the naming we should do it in the feature spec too, which I don't really want to do in this PR.

Reply:
Yeah sure, I do think they should both be changed but it's very minor and I'm happy to let it pass so we can land this PR 👍